### PR TITLE
:sparkles: PIC-1287: add privacy notice

### PR DESCRIPTION
**Pull request by katie-cawthorne**
- added link in footer to privacy notice 
- created privacy notice page
- added new end point for privacy page
- added new feature for privacy notice

### CODE REVIEW

File: server/views/privacy-notice.njk [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        about data protection. </p>
+
+    <p class="govuk-body govuk-!-margin-bottom-0">Information Commissioner's Office</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Wycliffe House</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Water Lane</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Wilmslow</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Cheshire</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">SK9 5AF</p>
+    <p class="govuk-body govuk-!-margin-bottom-0">Tel: 0303 123 1113</p>
+    <a class="govuk-link" href="www.ico.org.uk">www.ico.org.uk</a>
```
Comment (sarahyjja) [inline server/views/privacy-notice.njk:250]: The link here doesn't work. Maybe we should replace it with the https at the front?
a class="govuk-link" href="https://ico.org.uk/">www.ico.org.uk</a>

File: server/views/privacy-notice.njk [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">3rd Floor, Post Point 3.20
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">10 South Colonnades
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">London
+    </p>
+    <p class="govuk-body govuk-!-margin-bottom-0">E14 4PU
+    </p>
+    <a class="govuk-link" href="mailto:data.compliance@justice.gov.uk">data.compliance@justice.gov.uk
```
Comment (sarahyjja) [inline server/views/privacy-notice.njk:237]: I'm not sure but does all the links should have the size of the text or should they be a little smaller?

**Review by sarahyjja — CHANGES_REQUESTED** [review]
Very tiny changes about layout on the server/views/privacy-notice.njk :)

File: server/views/privacy-notice.njk [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    <h2 class="govuk-heading-m  govuk-!-margin-top-5">Changes to this policy
+    </h2>
+
+    <p class="govuk-body">We may change this privacy policy. Any changes to this privacy policy will apply to you and
+        your data immediately.</p>
+
+    <p class="govuk-body">If these changes affect how your personal data is processed, HMPPS Digital Services will take
+        reasonable steps to let you know.</p>
+
+    <p class="pac-secondary-text">Last updated: 8 April 2021</p>
```
[the remark this answers is not in the document]
Comment (katie-cawthorne) [reply server/views/privacy-notice.njk]: Good spot

File: server/views/privacy-notice.njk (hunk 1 above)
Comment (katie-cawthorne) [reply server/views/privacy-notice.njk]: Doh! Good spot, I'll update

File: server/views/privacy-notice.njk (hunk 2 above)
Comment (katie-cawthorne) [reply server/views/privacy-notice.njk]: Yeah you're right I need to add the govuk-body class, good spot 🙂

**Review by sarahyjja — APPROVED** [review]
coolio 😎

**Pull request merged**